composables/useLinks.ts

```diff
@@ -4,4 +4,5 @@ export const useLinks = () => ({
   discordUrl: "https://discord.gg/redot",
   twitterUrl: "https://x.com/redotengine",
   redditUrl: "https://www.reddit.com/r/RedotGameEngineMain/",
+  pressKit: "https://cdn.discordapp.com/attachments/1290096105094385695/1292789668907057292/Redot_Engine_-_Brand_Kit_v.1FINAL.zip?ex=6708f833&is=6707a6b3&hm=8d631f74ebb4bfc0e0b5de3989559a75820d42620944a2cc794ba4b60fbedaff&",
```

Comment:
It seems that this link has an expiration. I think we should download the zip file instead, and place it in the public folder. When press kit link is pressed, then it should download the file in the public folder